packages/compass-e2e-tests/helpers/commands/connect-form.ts

```diff
@@ -442,15 +442,15 @@ async function waitForElementAnimations(browser: CompassBrowser, element: any) {
 
 const colorMap: Record<string, string> = {
   'no-color': 'No Color',
-  color1: 'Red',
-  color2: 'Pink',
-  color3: 'Orange',
-  color4: 'Yellow',
-  color5: 'Green',
-  color6: 'Teal',
-  color7: 'Blue',
-  color8: 'Iris',
-  color9: 'Purple',
+  color1: 'Green',
```

Comment:
I can probably import this from usec-connection-color.ts or a common place rather than maintain another copy here, but I suppose it really isn't the end of the world if the E2E tests expect the colours to be what they are.

Reply:
it's actually not a bad thing to check for 👍